Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -13,54 +13,54 @@ Password Hashing
+the release of TYPO3 core version 9.
 
-This section is for administrators and users who want to know more about TYPO3 password hashing,
-have a basic understanding of the hashing algorithms and configuration in TYPO3.
+This section is for administrators and users who want to know more about TYPO3 password hashing and
+have a basic understanding of hashing algorithms and configuration in TYPO3.
 
 
 .. _password-hashing-basic-knowledge:
 
 Basic Knowledge
 ===============
 
-Storing plain text passwords to simply compare a stored password with a given user password when
-the user logs in has been a common attack vector in online systems ever since.
-The main risk is that some other system vulnerability or chain of vulnerabilities
-allows an attacker to download the frontend or backend user database table and then knows
-the plain text passwords of users. These passwords can then be used to log in as the specific
-user to get their TYPO3 instance specific privileges (for instance a backend admin)
-and - usually even worse - are often abused by attackers to try the same user / password combination
-at different services. Maybe the user uses the same login email and password for his bank account?
-
-To mitigate this risk, one-way `hash`_ algorithms have been invented: The given password is one-way
-transformed to some different string (a hash, also known as checksum), this hash is stored in the database
-instead of the plain text password. The idea is if you see the hash, you can not calculate back to the plain text
-password easily. That's why hashes are called "one-way": It's cheap to calculate a hash for a given
-password, but it is expensive (in terms of computation time) to determine a password for a given hash. If a user
-tries to log in and submits its password, the same one-way transformation is done again, if it is then identical
-with the hash stored in the database, the submitted password must have been correct and the login succeeds.
-
-The most well-known hash algorithm is MD5. Basic hash algorithms and especially MD5 have drawbacks tough:
-First, if you find some other string that resolves to the same hash, you are screwed (that's called
+If a database has been compromised and the passwords have been stored as plain text,
+the attacker has most likely gained access to more than just the user's TYPO3 Frontend / Backend account.
+It's not uncommon for someone to use the same email and password combination for more than one online service,
+such as their bank account, social media accounts, or even their email provider.
+
+To mitigate this risk, we can use one-way `hash`_ algorithms to transform a plain text password into an incomprehensible
+string of seemingly "random" characters.
+A hash, (also known as a checksum), is the result of a value (in this case the user's password) that has been
+mathematically calculated using a one-way function.
```

Comment:
I'd use "transformed" instead of "calculated". Otherwise it sounds like the password has been calculated. ;-)